pkg/git/detector/detector_whitebox_test.go

```diff
@@ -75,6 +91,22 @@ func XTestGitHubDetectorWithUsernameAndPassword(t *testing.T) {
 	printBuildEnvStats(buildEnvStats)
 }
 
+func XTestBitbucketDetectorWithUsernameAndPassword(t *testing.T) {
```

Comment:
This particular test is a bit tricky because Bitbucket doesn't have any `anonymous` user I could use as a username (as I can do in case of GitHub). When I leave username and password empty then the authentication fails. I can reach Bitbucket using an empty token and [this test is activated](https://github.com/redhat-developer/git-service/pull/10/files/7b74129fae90dbdd684cad3c038fb1bd6bd4535d#diff-8d3248024bbd50149e89d06034a64f4bR58) so we do have a test reaching the real service. 

Sorry for the `X ` prefix - it's an old habit from Gomega `¯\_(ツ)_/¯`

Reply:
The ignored test was skipped as part of the merge commit